frontend/widgets/OBSBasic.cpp

```diff
@@ -0,0 +1,1995 @@
+/******************************************************************************
+    Copyright (C) 2023 by Lain Bailey <lain@obsproject.com>
+                          Zachary Lund <admin@computerquip.com>
+                          Philippe Groarke <philippe.groarke@gmail.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#endif
```

Comment:
Marking same as the other `WIN32_LEAN_AND_MEAN` instances.

Reply:
See above.